projects/kit/components/input-date-range/input-date-range.component.ts

```diff
@@ -223,6 +227,14 @@ export class TuiInputDateRangeComponent
         return this.nativeFocusableElement ? this.nativeFocusableElement.value : '';
     }
 
+    get fillerLength(): number {
```

Comment:
These two can be readonly properties. That is if you need them in template, otherwise it could just be constants.

Reply:
i used constants in this component